<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorView</class>
 <widget class="QWidget" name="EditorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <layout class="QVBoxLayout" name="layout" stretch="1,0">
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QPlainTextEdit" name="textEdit"/>
   </item>
   <item>
    <widget class="QWidget" name="taskGroup" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="dueDateLabel">
        <property name="text">
         <string>D&amp;ue Date </string>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <property name="buddy">
         <cstring>dueDateEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="recurrenceCombo"/>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="doneLayout">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QCheckBox" name="doneButton">
          <property name="text">
           <string>Done</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="addAttachmentButton">
          <property name="toolTip">
           <string>Add Attachment...</string>
          </property>
          <property name="icon">
           <iconset theme="list-add">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeAttachmentButton">
          <property name="toolTip">
           <string>Remove Attachment</string>
          </property>
          <property name="icon">
           <iconset theme="list-remove">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="KDateComboBox" name="startDateEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="startTodayButton">
          <property name="text">
           <string>Select for &amp;Today</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="recurrenceLabel">
        <property name="text">
         <string>&amp;Recurrence</string>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <property name="buddy">
         <cstring>recurrenceCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KDateComboBox" name="dueDateEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="startDateLabel">
        <property name="text">
         <string>St&amp;art Date</string>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <property name="buddy">
         <cstring>startDateEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QListView" name="attachmentList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="flow">
         <enum>QListView::LeftToRight</enum>
        </property>
        <property name="isWrapping" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionEditor">
   <property name="text">
    <string>Editor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KDateComboBox</class>
   <extends>QComboBox</extends>
   <header>KDateComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
